src/libraries/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -291,7 +291,7 @@ public static void ParseJson_UnseekableStream(bool compactData, TestCaseType typ
                     new WrappedMemoryStream(canRead: true, canWrite: false, canSeek: false, bytes)));
         }
 
-        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [Theory]
```

Comment:
Does this mean this test is fine?

Reply:
Yes, the test didn't fail locally when building wasm tests from this [commit hash](https://github.com/dotnet/runtime/commit/fbf84d6911bf26f1bdf32a5071d50b449927f210). The first commit reverts all changes made in #37822, regardless of whether they fail or pass, and the third commit re-adds the tests that still fail after that [commit hash](https://github.com/dotnet/runtime/commit/fbf84d6911bf26f1bdf32a5071d50b449927f210)